packages/code-connect/components/ExpandableSection/ExpandableSection.figma.tsx

```diff
@@ -0,0 +1,158 @@
+import figma from '@figma/code-connect';
+import { ExpandableSection, ExpandableSectionToggle, Stack, StackItem } from '@patternfly/react-core';
+
+// TODO: DESIGN: This component needs to be overhauled. Using the base component approach present in
+// other components would significantly reduce complexity.
+// TODO: DESIGN: Create toggle component
+// onToggle={() => {}} // only required if the user wants to be able to have other side effects when opening/closing
+// isExpanded={() => {}} // only required if the user wants to be able to have other side effects when opening/closing
+
+// Documentation for ExpandableSection can be found at https://www.patternfly.org/components/expandable-section
+
+const customToggleContent = `
+    <div>
+    <span>You can also use icons </span>
+    <CheckCircleIcon />
+    <span> or badges </span>
+    <Badge isRead={true}>4</Badge>
+    <span> !</span>
+  </div>
+`;
+
+figma.connect(
+  ExpandableSection,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=2404-21',
+  {
+    props: {
+      toggleText: figma.string('Toggle Text More')
+    },
+    example: (props) => (
+      <ExpandableSection
+        onToggle={() => {}}
+        toggleText={isExpanded ? 'Show less basic example content' : `${props.toggleText}`}
+      >
```

Comment:
stub out `isExpanded={false/true}` doesn't need to be connected to figma, `true/false` are acceptable values.